lib/model/query/audits.js

```diff
@@ -110,5 +110,56 @@ const getBySubmissionId = (submissionId, options) => ({ all }) =>
   _getBySubmissionId(all, options, submissionId);
 
 
-module.exports = { log, getLatestByAction, get, getBySubmissionId };
+
+const _getByEntityId = (fields, options, entityId) => sql`
+SELECT ${fields} FROM audits
+  LEFT JOIN actors ON actors.id=audits."actorId"
+  LEFT JOIN (
+    SELECT submissions.*, submission_defs."userAgent" FROM submissions
+    JOIN submission_defs ON submissions.id = submission_defs."submissionId" AND root -- just for the sake of userAgent which we don't need, we should remove it
+  ) submissions ON (audits.details->'submissionId')::INTEGER = submissions.id AND action IN ('entity.create', 'entity.update.version')
```

Comment:
What do you think about joining in `submissions` just for `entity.create`, not `entity.update.version`? At least right now, submissions can't update entities, so I think the only action that's really connected to submissions is `entity.create`.